### ISSUE TITLE
How do I use TextureUV with multiple textures

### ISSUE THREAD

**Issue by lishi0927** [facebookresearch/pytorch3d#1017]
Dear Authors,
I'm glad to use Pytorch3d for my research about differentiable rendering. But now I'm confusion about how to use TextureUV with multiple textures. 
I  want to train two textures with our custom shader, for example, lightmap and albedomap. I use these two textures in our custom shader to fit the ground truth images, But I find TextureUV supports only  one image, TextureAtlas supports  multiple textures but it can't be visited through coordinate.
Could you give me some suggestions to acquire these two textures through neural network?
Thank you.

**Comment by gkioxari** [comment]
TexturesUV supports only one image for now.

**Comment by lishi0927** [comment]
Thank you for your reply.
I have two ideas about this enhancement. One is that we can write a new type that combine textureatlas and textureuv, The other is modify the Mesh class to support texture lists.
Could you give me some suggestions about this realization, which one would be better or you have best plans?

**Comment by bottler** [comment]
Both TexturesAtlas and TexturesUV support batches, but these correspond to the batch of meshes. So actually I don't think looking at TexturesAtlas will help.

I think an easy solution would be to create a new TexturesBase subclass which owns a list of TexturesBase objects. Its sample_textures would call sample_textures on each element of the list and concatenate the results along the channel axis. This requires little or no change to existing classes.

(If the UV coordinates are shared then you can use a single mega TexturesUV with many channels.)
